structs.wdl

```diff
@@ -0,0 +1,32 @@
+version 1.0
+
+import "tasks/bwa.wdl" as bwa
+import "tasks/common.wdl" as common
+
+struct Readgroup {
+    String id
+    File R1
+    String? R1_md5
+    File? R2
+    String? R2_md5
+}
+
+struct Library {
+    String id
+    Array[Readgroup]+ readgroups
+}
+
+struct Sample {
+    String id
+    Array[Library]+ libraries
+}
+
+struct Root {
+    Array[Sample] samples
+}
+
+struct GamsInputs {
+    Boolean? combineReads
```

Comment:
Please make this `Boolean combineReads = false`, that will allow you to remove all the select_first function calls

Reply:
Structs can't have defaults